nss-day-cohort-30/bangazon-site-mighty-mahi-mahi
# initial setup done

Fixes:
- created initial project and database
-

Changes proposed in this request & reasons behind organizational or architectural decisions:
-
-

Steps to test:
- update-database in the Package Manager Console
-
-
-

System configuration (3rd party libraries to be installed, command line utilities to run, UI instructions, expected outcome):
-
-

Commit message:
- initial setup done

Link to feature ticket:
-

@mighty-mahi-mahi
